itonx/ShelterVault
# Windows 11 UI like

## Summary 
- Refactored code to use behaviors.
- Reorganized folder structure, namespaces and class names.
- Added NavigationView.
- Added page transitions on CredentialsView commands.
- Added theme changer at runtime.
- Added support for Light and Dark themes.
- Added Neuromancer theme.
- Removed unused code.
- Added services and DI.
- Added custom TitleBar with interactive controls.
- Refactored resource dictionaries to reuse styles and support themes.

## Testing
- Create new master key.
- Confirm master key.
- Add new credentials.
- Update existing credentials.
- Delete credentials.
- Verify Pending Changes message is displayed when new credentials are being added/updated/deleted.
- Navigation between pages.
- Change themes at runtime.
- Confirm button theme changer is an interactive control in the Title Bar.
